File: 01/main.cpp
#include <fstream>
#include <iostream>
#include <sstream>
#include <string>

int main() {
  std::size_t max_elf_cals{}, current_elf_cals{}, cal_num;
  std::ifstream input_file_stream{"input.txt"};
  std::string cal;

  while (std::getline(input_file_stream, cal)) {
    if (cal == "") {
      if (current_elf_cals > max_elf_cals) {
        max_elf_cals = current_elf_cals;
      }
      current_elf_cals = 0;
    } else {
      std::stringstream cal_stream{cal};
      cal_stream >> cal_num;
      current_elf_cals += cal_num;
    }
  }

  std::cout << max_elf_cals << '\n';
}

File: 01/main.py
with open("input.txt") as f:
    cals = [l.strip() for l in f]

max_elf_cals = current_elf_cals = 0

for cal in cals:
    if cal == "":
        if current_elf_cals > max_elf_cals:
            max_elf_cals = current_elf_cals
        current_elf_cals = 0
    else:
        current_elf_cals += int(cal)

print(max_elf_cals)
